app/helpers/gov_one_helper.rb

```diff
@@ -0,0 +1,49 @@
+module GovOneHelper
+  # @return [URI]
+  def login_uri
+    params = {
+      redirect_uri: GovOneAuthService::CALLBACKS[:login],
+      client_id: Rails.application.config.gov_one_client_id,
+      response_type: 'code',
+      scope: 'email openid',
+      nonce: SecureRandom.uuid,
+      state: SecureRandom.uuid,
+    }
+
+    session[:gov_one_auth_state] = params[:state]
+
+    gov_one_uri(:login, params)
+  end
+
+  # @return [URI]
+  def logout_uri
+    params = {
+      post_logout_redirect_uri: GovOneAuthService::CALLBACKS[:logout],
```

Comment:
I have no issue locally, so I am now going to retest in prod @jack-coggin

Reply:
OK, will retest and revert

Reply:
Right OK, thanks